extensions/interactions/NumberWithUnits/directives/number_with_units_help_modal_directive.html

```diff
@@ -0,0 +1,65 @@
+<div class="modal-header">
+  <b>Rules and possible unit formats</b>
+</div>
+<div class="modal-body">
+  <style>
+    td {
+      padding: 5px;
+      width: 30%;
+    }
+  </style>
+  <table class="table-bordered">
+    <tr>
```

Comment:
Set heading to something like `Conventions` or `Writing guidelines`.

Reply:
Ok, `Unit writing guidelines` seems better.

Reply:
Done!